javascript/signin.js

```diff
@@ -0,0 +1,67 @@
+import { accountInfo } from "./account.js"
+import { login } from "./api/auth/login.js"
+import { passwordToggle } from "./passwordToggle.js"
+import { emailFocus, emailBlur, passwordFocus, passwordBlur } from "./signFunction.js"
+
+// 
+const inputEmail = document.querySelector('.email-input')
+const inputPassword = document.querySelector('.password-input')
+const passwordEncryptButtonList = document.querySelector('.eye-btn')
+const errorEmailMessage = document.querySelector('.error-email-message')
+const errorPasswordMessage = document.querySelector('.error-password-message')
+const signButton = document.querySelector('.sign-btn')
+
+// 토큰이 존재한다면 folder.html 이동
+const tokenExistence = localStorage.getItem('accessToken')
+tokenExistence ? window.location.assign('folder.html') : ''
+
+// 로그인 버튼, 경우에 따른 케이스 적용
+const onSubmit = (e) => {
+  e.preventDefault()
+
+  const inputEmailValue = inputEmail.value
+  const inputPasswordValue = inputPassword.value
+  const findEmail = accountInfo.find((item) => item.email === inputEmailValue)?.email
+  const findPassword = accountInfo.find((item) => item.password === inputPasswordValue)?.password
+  const matchedEmail = inputEmailValue === findEmail
+  const matchedPassword = inputPasswordValue === findPassword
+
+  const inputErrorHandler = (inputType, errorMessage, type) => {
+    if (type === 'email') {
+      inputType.classList.add('input-error')
+      errorEmailMessage.textContent = errorMessage
+      return
+    }
+    inputType.classList.add('input-error')
+    errorPasswordMessage.textContent = errorMessage
+  }
```

Comment:
해당 함수가 onSubmit 함수안에 잇을 필요가 있을까요?

Reply:
아래처럼도 가능합니다. 참고만해주세요~

```suggestion
  const inputErrorHandler = (inputType, errorMessage, type) => {
    const errorEl = type === "email" ? errorEmailMessage : errorPasswordMessage;

    inputType.classList.add("input-error");
    errorEl.textContent = errorMessage;
  }
```

Reply:
참고하여서 분리하도록 하겠습니다. 감사합니다!!